spring-kafka/src/test/java/org/springframework/kafka/listener/KafkaMessageListenerContainerTests.java

```diff
@@ -237,7 +243,7 @@ public void testSlowConsumerCommitsAreProcessed() throws Exception {
 		// Verify that commitSync is called when paused
 		assertThat(latch.await(60, TimeUnit.SECONDS)).isTrue();
 		verify(consumer, atLeastOnce()).pause(any(TopicPartition.class), any(TopicPartition.class));
-		verify(consumer, atLeastOnce()).commitSync(any());
+		verify(consumer, atMost(2)).commitSync(any());
```

Comment:
Doh... Good catch @mbogoevici !

This code has confused me:
```
if (foundSize > maxNumberOfInvocations) {
        new Reporter().wantedAtMostX(maxNumberOfInvocations, foundSize);
}
```

I haven't looks into the logic closely and somehow my English knowledges let me down :cry: 

I definitely meant `atLeast(2)`.

The previous `atLeastOnce()` isn't good for us because we always call it once in the `onPartitionsAssigned()` anyway.

Fee free to fix it in your branch! No need separate PR or even commits, like Boot guys requie :wink: 
I will understand you properly.